com.github.quaternion.json

```diff
@@ -0,0 +1,21 @@
+{
+    "id": "com.github.quaternion",
+    "branch": "master",
+    "rename-icon": "quaternion",
+    "rename-desktop-file": "quaternion.desktop",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "quaternion",
+    "tags": ["nightly"],
```

Comment:
For now, Flathub is just for released versions of software rather than nightly builds, you should remove this.